lte/gateway/docker/s1ap/readme.md

```diff
@@ -0,0 +1,156 @@
+# Build and publish s1aptester images
+
+1. Clone the magma repository and move into the AGW docker directory in the repo and run build script.
+
+```
+git clone https://github.com/magma/magma
+cd magma/lte/gateway/docker
+s1ap/build-s1ap.sh
+```
+
+2. Publish images to your registry
+
+```
+s1ap/publish-s1ap.sh yourregistry.com/yourrepo/
+```
+
+# Deploy containerized S1APTester
+
+## Create your environment
+
+### AWS Environment
+
+* VPC (vpc-s1ap) with a CIDR of 192.168.0.0/16
+* Internet Gateway (igw1) attached to VPC (vpc-s1ap)
+* Route table (rt1) with a default gateway to the Internet Gateway (igw1)
+* 3 Subnets with an associated Route Table (rt1)
+  * SGi: 192.168.60.0/24 (subnet-sgi)
+  * S1: 192.168.128.0/23 (subnet-s1)
+  * S1AP Management: 192.168.59.0/24 (subnet-management)
+* Import/Create Key Pair (kp1)
+
+### AGW instance
+
+* Ubuntu 20.04 x86
+* t2.large
+* Security group
+  * Allow SSH from My IP, or private address of your bastion host. **Important to not allow Public SSH access**
+  * Allow all traffic from 192.168.60.0/24
+  * Allow all traffic from 192.168.128.0/23
+* 3 Interfaces
+  * eth0: 192.168.129.1 (subnet-s1)
+    * Add Public IP
+  * eth1: 192.168.60.142 (subnet-sgi)
+  * eth2: Auto-assign (subnet-management)
+* 50 GB Disk
+* Key Pair (kp1)
+
+### s1aptester instance
+
+* Ubuntu 20.04 x86
+* t2.large
+* Security group
+  * Allow SSH from My IP, or private address of your bastion host. **Important to not allow Public SSH access**
+  * Allow all traffic from 192.168.60.0/24
+  * Allow all traffic from 192.168.128.0/23
+* 3 Interfaces
+  * eth0: Auto-assign (subnet-management)
+    * Add Public IP
+  * eth1: 192.168.60.141 (subnet-sgi)
+  * eth2: 192.168.128.11 (subnet-s1)
+* 50 GB Disk
+* Key Pair (kp1)
+
+### Traffic Generator instance
+
+* Ubuntu 20.04 x86
+* t2.large
+* Security group
+  * Allow SSH from My IP, or private address of your bastion host. **Important to not allow Public SSH access**
+  * Allow all traffic from 192.168.60.0/24
+  * Allow all traffic from 192.168.128.0/23
+* 3 Interfaces
+  * eth0: Auto-assign (subnet-management)
+    * Add Public IP
+  * eth1: 192.168.60.144 (subnet-sgi)
+  * eth2: 192.168.129.42 (subnet-s1)
+* 50 GB Disk
+* Key Pair (kp1)
+
+## Add your ssh key pair (kp1) used when creating the instances to your ssh-agent
+
+```
+ssh-add .ssh/id_rsa
+ssh-add -l
+3072 SHA256:KWaiSLPiTYy5tRGXSKpkuwJcYDGy92NTMi0v3khLjJY (RSA)
+```
+
+## Run S1APTester playbook
+
+### ssh to the agw instance and deploy as a containerized AGW
+
+```
+# ssh to AGW instance
+ssh -A ubuntu@184.72.47.28
+
+# Download containerized AGW bootstrap script
+wget https://raw.githubusercontent.com/magma/magma/master/lte/gateway/deploy/agw_install_docker.sh
+
+# Create certs directory and populate with your orc8r rootCA
+sudo mkdir -p /var/opt/magma/certs
+sudo su root -c 'sudo echo "-----BEGIN CERTIFICATE-----
+-----END CERTIFICATE-----" > /var/opt/magma/certs/rootCA.pem'
```

Comment:
It would be good to clarify here, that the content of the rootCA was omitted in this documentation.